EventFilter/RPCRawToDigi/interface/RPCLBRecord.h

```diff
@@ -0,0 +1,71 @@
+#ifndef EventFilter_RPCRawToDigi_RPCLBRecord_h
+#define EventFilter_RPCRawToDigi_RPCLBRecord_h
+
+#include <stdint.h>
```

Comment:
Within EventFilter only <stdint.h> is used, with uint* in the global namespace.  Could you explicitly confirm I should start using <cstdint> and std::uint* before I make the changes?